### PR TITLE
fix: responsive designing

### PR DESCRIPTION
**Pull request by amitojsingh**
Screenshot 2024-04-26 at 4 00 22 PM
Screenshot 2024-04-26 at 3 59 33 PM
Screenshot 2024-04-26 at 3 59 08 PM
Screenshot 2024-04-26 at 3 59 00 PM
Screenshot 2024-04-26 at 3 58 31 PM
Screenshot 2024-04-26 at 3 58 19 PM
Screenshot 2024-04-26 at 3 58 07 PM

### CODE REVIEW

File: src/components/gurfateh/Gurfateh.tsx [hunk 1]
```diff
@@ -5,13 +5,13 @@ export default function Gurfateh() {
   const { t: text } = useTranslation();
 
   return (
-    <main className='relative flex lg:w-2/5 w-2/3 justify-center xl:justify-start self-center'>
-      <div className='bg-slate-100/75 shadow rounded-3xl flex-col text-left gap-4 '>
-        <div className='flex-grow w-2/3 m-auto my-2 p-3'>
-          <h1 className='text-xl font-medium text-black text-left gurmukhi'>
```
Comment (MySmilingTurban) [inline src/components/gurfateh/Gurfateh.tsx:11]: gurmukhi classname is needed to apply our chosen font for Gurmukhi text

File: src/components/ssa/index.tsx [hunk 2]
```diff
@@ -6,9 +6,9 @@ export default function Ssa({ name }: { name: string }) {
   const { t: text } = useTranslation();
   return (
     <div>
-      <p className="text-3xl">
-        <span className="font-medium gurmukhi">{text('SSA_PUNJABI')}</span>{' '}
-        <span className="text-3xl text-slate-500 font-serif">
+      <p className='text-2xl lg:text-3xl text-[#112D3D]'>
+        <span className='font-medium gurmukhi'>{text('SSA_PUNJABI')}</span>{' '}
+        <span className='cormont-garamond-light capitalize'>
```
Comment (MySmilingTurban) [inline src/components/ssa/index.tsx:11]: we should discuss this in a meeting with Inder Singh Veerji to see which font suits well
(I love how this new font adds more flavour to the app)

File: src/components/gurfateh/Gurfateh.tsx (hunk 1 above)
Comment (amitojsingh) [reply src/components/gurfateh/Gurfateh.tsx:11]: fixed

File: src/components/ssa/index.tsx (hunk 2 above)
Comment (amitojsingh) [reply src/components/ssa/index.tsx]: fixed

File: src/styles/globals.scss [hunk 3]
```diff
@@ -2,6 +2,8 @@
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
 
+@import url('https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&display=swap');
+
 @font-face {
```
Comment (MySmilingTurban) [inline src/styles/globals.scss:7]: nit: remove redundant font

**Review by MySmilingTurban — CHANGES_REQUESTED** [review]
all looks good, just remove the import statement of the font not in use anymore

File: src/styles/globals.scss (hunk 3 above)
Comment (MySmilingTurban) [reply src/styles/globals.scss]: fixed

**Review by MySmilingTurban — APPROVED** [review]
Looks good to me!

**Pull request merged**